sdk/webpubsub/azure-messaging-webpubsub/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.3.0-beta.1 (Unreleased)
 
+- Added a `clientEndpointType` option to `GenerateClientTokenOptions` to specify the type of client endpoint
+  when generating token. This option can be used to generate token and client connection URL for a specific client endpoint type, such as `Default` or `MQTT`.
 - Migrated serialization to `azure-json` which offers implementation agnostic serialization, providing support for
   more serialization frameworks than just Jackson.
```

Comment:
Put these in `### Features Added` section

Also, do you need to mention the added API addConnectionsToGroups?

Reply:
Yeah I need to do that, will update it now